charts/rancher-istio/1.8.5000/values.yaml

```diff
@@ -39,32 +39,6 @@ gateways:
       app: istio-ingressgateway
```

Comment:
labels are also configured in sub charts，look like can be removed.

Reply:
updated